docs/release-notes/sso-auto-redirect.rst

```diff
@@ -0,0 +1,14 @@
+:orphan:
+
+** New Features**
+
+- Master Configuration: Add an ``always_redirect`` option to OIDC and SAML configurations. When enabled, this option
+- ```
```

Comment:
remove the ```

Reply:
Adjusted! It looks like this got added in from github automation for the earlier suggestions that I hadn't caught